from program import run

result = run("Egor")

# Если указать неверные аттрибуты или ожидаемое значение, то будет высвечиваться ошибка
result2: int = run(25)